proto/stride/stakeibc/tx.proto

```diff
@@ -314,3 +316,26 @@ message MsgUpdateTradeRoute {
   ];
 }
 message MsgUpdateTradeRouteResponse {}
+
+// Registers a community pool rebate by specifying the amount of the liquid
+// stake
+message MsgRegisterCommunityPoolRebate {
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stride/x/stakeibc/MsgRegisterCommunityPoolRebate";
+
+  // Message signer
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+  // Chain ID of the community pool
+  string chain_id = 2;
+  // Rebate percentage (e.g. 0.5 for 50%)
+  string rebate_percentage = 3 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  // Magnitude of the community pool liquid stake associated with the rebate
```

Comment:
suggest: number of native tokens liquid staked by the community pool

Reply:
Yes I definitely agree you should make clear this liquid staked amount is in the native token, not the staked token or the revenue token.

Reply:
done

Reply:
lgtm